CONTRIBUTING.md

```diff
@@ -407,17 +373,169 @@ MyClass myThirdClass = MyClass::String("abc") ;
+by following a procedure similar to the one described in the [Development Dockerfile](./docker/development/Dockerfile).
+
+The GNU Make tool is used to control the generation of executable in order to build OSTk docker containers and libraries. All the make recipes that are used can be found in the `Makefile` in the project level directory of each OSTk library.
+
+To start the development environment:
+
+```shell
+make start-development
+```
+
+This will:
+
+1. Build the `openspacecollective/open-space-toolkit-astrodynamics-development` Docker image.
+2. Create a development environment container with local source files and helper scripts mounted.
+3. Start a `bash` shell from the `./build` working directory.
+
+If developing in VSCode, we recommend developing inside a VSCode Dev Container, which can be done according to the documentation [here](https://code.visualstudio.com/docs/devcontainers/containers). Simply hit Ctrl+Shift+P and type `Dev Containers: Reopen in Dev Container` to launch the development environment. This allows you to Ctrl+Click on classes and be taken to their source code in C++.
+
+####  6.1.1. <a name='DevelopmentAcrossOSTkLibraries'></a>Development Across OSTk Libraries
+Please note that the development environment above only allows you to build an executable from the current ostk repo you are working on, not the other ostk repos that are part of the whole library. If you would like to do that, you can run the make recipe below which will mount another ostk library into your development image so you can make changes to multiple ostk repos at a time and test them out together.
+```shell
+make start-developlepment-link links="/your/path/to/another/ostk/repo/you/would/to/mount"
+```
+
+###  6.2. <a name='Build'></a>Build
+
+From the `./build` directory:
+
+```shell
+cmake ..
+make
+```
+
+*Tip: `helpers/build.sh` simplifies building from within the development environment.* \
+
+*Tip: `helpers/debug.sh` allows you to debug your code from within the development environment using `gdb`.* \
+
+*Tip: The `ccmake` tool is installed in the dev container and can be used to configure CMake project options to speed up your build time. It can be run with the `cmake ../` command from the `./build` directory*
+
+*Tip: `helpers/clean.sh` cleans the entire build directory.* \
+
+###  6.3. <a name='Test'></a>Test
+
+*Tip: `helpers/test.sh` simplifies running C++ tests from within the development environment. The name of the desired test to be run can be added afterwords according to documentation [here.](http://google.github.io/googletest/advanced.html#running-a-subset-of-the-tests)*
+
+###  6.4. <a name='Python'></a>Python
+
+*Tip: `helpers/install-python.sh` installs the currently build shared object libraries as a python package within the development environment so that you can use your most recent changes to OSTk in python.
```

Comment:
```suggestion
*Tip: `helpers/install-python.sh` installs the currently build shared object libraries as a python package within the development environment so that you can use your most recent changes to OSTk in python.*
```